src/redux/modules/producers/producers.js

```diff
@@ -131,3 +138,11 @@ export function create(data) {
     promise: (client) => client.post('/producers', { data }),
   };
 }
+
+/**
+ * After a new producer is created we reset
+ * in the store `createdProducer` key
+ */
+export function resetCreated() {
+  return { type: RESET_CREATED_PRODUCER };
+}
```

Comment:
On producer created we set that producer object on `producerCreated` to be used after that for supplier creation (we need producer ID). This action is for cleaning redux state when we finish